lti_placements/canvas_placements.py

```diff
@@ -248,11 +253,11 @@ def main() -> Sequence[DataSourceStatus]:
     This method is invoked when its module is executed as a standalone program.
     '''
 
+    # Get ids for tools in lti_type table as supported tools
```

Comment:
Nit-pick: This comment seems a little out of place here. I understand that you're explaining why the `SUPPORTED_LTI_TOOLS` dropped out, but once that's gone, there won't be much context for this. I'd put this wherever you call `get_supported_lti_tools`, or delete it.

Reply:
Yeah that should have been removed